config/samples/xgboost-dist/predict.py

```diff
@@ -29,7 +29,8 @@ def predict(args):
     dmatrix, y_test = read_predict_data(rank, world_size, None)
 
     model_path = args.model_path
-    booster = read_model("oss", model_path, args)
+    storage_type = args.model_storage_type
```

Comment:
can you check the storage_type here ? local or oss ? if users define other types, we need to raise an exception here.

Reply:
I guess some refactoring can be done so that the args checking can be done in main.py once and other functions are invoked if there is any problem with args.

Reply:
The storage type is checked in the dump_model function in utils.py file, which handles raising the exception when the storage type is given other than "oss" or "local"

Reply:
done